README.md

```diff
@@ -48,7 +50,19 @@ On Fedora systems, the following command will install these dependencies :
 sudo dnf install alsa-lib-devel make gcc
 ```
 
-Once you've cloned this repository you can build *librespot* using `cargo`.
+librespot currently offers the a selection of [audio backends](https://github.com/librespot-org/librespot/wiki/Audio-Backends).
+```
+Rodio (default)
+ALSA
+PortAudio
+PulseAudio
+JACK
+SDL
+Pipe
+```
+Please check the corresponding [compiling wiki entry](https://github.com/librespot-org/librespot/wiki/Compiling#dependencies) for backend specific dependencies.
```

Comment:
Or at the very least this should link to COMPILING. I _think_ relative links are supported.

Reply:
Fixed.